### PR TITLE
Update datadog_monitor.py

### PR DESCRIPTION
**Pull request by bbadisa**
Updated the Name, Escalation Message Variables with fix_template_vars methods.

##### ISSUE TYPE

 - Feature Pull Request
 - New Module Pull Request
 - Bugfix Pull Request
 - Docs Pull Request

##### ANSIBLE VERSION

```

```

##### SUMMARY

```

```

### CODE REVIEW

**Comment by bbadisa** [comment]
HI,


I have modified this modue for using this curly braces {{  which will be as [[ in playbook, so modified the  name variable and escalation message variable and also modified the module with all the variable which name and escalation message.

File: lib/ansible/modules/monitoring/datadog_monitor.py [hunk 1]
```diff
@@ -274,7 +274,8 @@ def install_monitor(module):
         "no_data_timeframe": module.params['no_data_timeframe'],
         "timeout_h": module.params['timeout_h'],
         "renotify_interval": module.params['renotify_interval'],
-        "escalation_message": module.params['escalation_message'],
+        #"escalation_message": module.params['escalation_message'],
```
Comment (skornehl) [inline lib/ansible/modules/monitoring/datadog_monitor.py]: Please remove commented out line

File: lib/ansible/modules/monitoring/datadog_monitor.py [hunk 2]
```diff
@@ -286,6 +287,7 @@ def install_monitor(module):
         options["thresholds"] = module.params['thresholds']
 
     monitor = _get_monitor(module)
+    #print monitor
```
Comment (skornehl) [inline lib/ansible/modules/monitoring/datadog_monitor.py]: Please remove commented out line

**Comment by skornehl** [comment]
Thanks! 
shipit

**Comment by kaarolch** [comment]
@bbadisa  - could you also update the documentation for this module. The name part with updated documentation is already in one of my [PR](https://github.com/ansible/ansible/pull/19377)

**Comment by bbadisa** [comment]
Hi Team,
May i know, what is left more on this..

**Comment by jctanner** [comment]
re-running CI

**Comment by maxamillion** [comment]
Closing as per https://github.com/ansible/ansible/pull/38483

**Pull request closed without merging**